plugins/modules/computer.py

```diff
@@ -56,11 +56,13 @@
 seealso:
 - module: microsoft.ad.domain
 - module: microsoft.ad.domain_controller
-- module: microsoft.ad.domain_membership
+- module: microsoft.ad.membership
 - module: microsoft.ad.object_info
 - module: microsoft.ad.object
 - module: microsoft.ad.offline_join
-- module: community.windows.win_domain_group
+- ref: Migration guide <ansible_collections.microsoft.ad.docsite.guide_migration.migrated_modules.win_domain_computer>
```

Comment:
Should the note I added be remove here in favour of the seealso or having both is fine? Same for the other modules.

Reply:
Oh good question, I didn't actually look closely at the module docs yet. For now I'd say let's have both maybe, might stand out in different ways. Can always remove later.